app/services/events/stores/postgres_store.rb

```diff
@@ -333,6 +331,24 @@ def group_scope(scope)
         scope.where('events.properties @> ?', { group.parent.key.to_s => group.parent.value }.to_json)
       end
 
+      def filters_scope(scope)
+        matching_filters.each do |key, value|
+          scope = scope.where(
+            'events.properties @> ?',
+            { key.to_s => value }.to_json
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)